docs/pages/reference/mesheryctl-commands.md

```diff
@@ -831,23 +831,29 @@ Installation, troubleshooting and debugging of Meshery and its adapters
 <thead>
   <tr>
     <th>Command</th>
-    <th>Subcommand</th>
-    <th>Flag</th>
+    <th>Subcommand 1</th>
+    <th>Subcommand 2</th>
```

Comment:
@alphaX86  what do you think?

Reply:
Will break the heading consistency

Reply:
IMO, I think we can just remove the `exp` from table (we can highlight from the heading before table itself) so we can just put it as Command and Subcommand. // @dusdjhyeon